android/app/build.gradle

```diff
@@ -151,6 +152,9 @@ android {
 }
 
 dependencies {
+    // Multidex support for < Android 5.0 (API level 21)
```

Comment:
Should this be greater than instead of less than?

Reply:
Nope, this library is not required for newer versions.